deploy/template.yaml

```diff
@@ -2216,6 +2540,180 @@ Resources:
       Threshold: 1
       ComparisonOperator: GreaterThanOrEqualToThreshold
 
+  ## PartialNameMatchHandler
+  PartialNameMatchFunction:
+    Type: AWS::Serverless::Function
+    DependsOn: PartialNameMatchFunctionLogGroup
+    Properties:
+      FunctionName: !Sub "BAV-PartialNameMatch-${AWS::StackName}"
+      Handler: PartialNameMatchHandler.lambdaHandler
+      CodeUri: ../src/
+      ReservedConcurrentExecutions: !If
+        - ApplyReservedConcurrency
+        - !Ref LambdaConcurrency
+        - !Ref AWS::NoValue
+      Environment:
+        Variables:
+          PARTIAL_MATCHES_BUCKET: !Ref PartialNameMatchBucket
+          PARTIAL_MATCHES_BUCKET_KEY: !Ref PartialNameMatchBucketKeyAlias
+          POWERTOOLS_SERVICE_NAME: "PartialNameMatchHandler"
+      Policies:
+        - AWSLambdaBasicExecutionRole
+        - AWSXrayWriteOnlyAccess
+        - S3WritePolicy:
+            BucketName: !Ref PartialNameMatchBucket
+        - Statement:
+            - Sid: "KMSEncryptForPartialMatchBucket"
+              Effect: Allow
+              Action:
+                - "kms:Encrypt"
+                - "kms:GenerateDataKey"
+              Resource: !GetAtt PartialNameMatchBucketKey.Arn
+        - Statement:
+            - Sid: "KMSDecryptForPartialMatchQueue"
+              Effect: Allow
+              Action:
+                - "kms:Decrypt"
+                - "kms:GenerateDataKey"
+              Resource: !GetAtt PartialMatchesEncryptionKey.Arn
+        - Statement:
+            - Effect: Allow
+              Action:
+                - "sqs:SendMessage"
+              Resource: !GetAtt TxMASQSQueue.Arn
+            - Effect: Allow
+              Action:
+                - "kms:Encrypt"
+                - "kms:GenerateDataKey"
+              Resource: !GetAtt TxMAKMSEncryptionKey.Arn
+      Events:
+        SQSTrigger:
+          Type: SQS
+          Properties:
+            Queue: !GetAtt PartialMatchesQueue.Arn
+            BatchSize: 1
```

Comment:
I thought our batchSize was going to be 10, or was that something else?

Reply:
Nope something I missed in rebase, will revert back to 10

Reply:
Changed to 10